homeassistant/components/qnap/sensor.py

```diff
@@ -375,23 +374,23 @@ def native_value(self):
         """Return the state of the sensor."""
         free = float(self.coordinator.data["system_stats"]["memory"]["free"]) / 1024
         if self.entity_description.key == "memory_free":
-            return round_nicely(free)
+            return free
 
         total = float(self.coordinator.data["system_stats"]["memory"]["total"]) / 1024
 
         used = total - free
         if self.entity_description.key == "memory_used":
-            return round_nicely(used)
+            return used
 
         if self.entity_description.key == "memory_percent_used":
-            return round(used / total * 100)
+            return used / total * 100
 
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
         if self.coordinator.data:
             data = self.coordinator.data["system_stats"]["memory"]
-            size = round_nicely(float(data["total"]) / 1024)
+            size = float(data["total"]) / 1024
```

Comment:
This makes no sense, the suggested_display_precision does not apply to attributes?

Reply:
Mistake, corrected